src/components/dashboard/SendLinkSummary.js

```diff
@@ -193,25 +190,11 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
       const desktopShareLink = generateSendShareText(paymentLink, amount, counterPartyDisplayName, profile.fullName)
 
       // Show confirmation
-      screenProps.push('SendConfirmation', {
+      push('TransactionConfirmation', {
         paymentLink: desktopShareLink,
-        amount,
-        reason,
-        counterPartyDisplayName,
       })
     }
-  }, [
-    generateSendShareText,
-    generateSendShareText,
-    setLink,
-    canShare,
-    link,
-    amount,
-    reason,
-    counterPartyDisplayName,
-    profile,
-    screenProps,
-  ])
+  }, [generateSendShareText, setLink, canShare, link, amount, counterPartyDisplayName, profile, push])
```

Comment:
probably here you also can avoid duplicating long deps list as i proposed in `ReceiveSummary.js`

Reply:
done